stacks_queues/queue.py

```diff
@@ -23,39 +23,64 @@ def enqueue(self, element):
             In the store are occupied
             returns None
         """
-        pass
+        if self.front == -1:
+            self.front = self.rear = 0
+
+        if self.full():
```

Comment:
💫 Very nice modification to increase the buffer size instead of raising an exception